### PR TITLE
Fixes #8625: There is a /var/cfengine/inputs on a Rudder server

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/8625

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-init [hunk 1]
```diff
@@ -297,17 +306,23 @@ echo
...
+then
+  echo -n "Configuring and installing initial CFEngine promises..."
+  rm -rf /var/cfengine/inputs/*
+  cp -r $TMP_DIR/community/* /var/cfengine/inputs/
+  echo " Done."
+fi
+
 # Configure initial promises
-echo -n "Configuring and installing initial CFEngine promises..."
+echo -n "Configuring and installing initial Rudder policies..."
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-init]: \o/

File: rudder-webapp/SOURCES/rudder-init [hunk 2]
```diff
@@ -297,17 +306,23 @@ echo
 # Leave the user a chance to stop only if there is a user
 tty >/dev/null && Pause
 
+# Replace CFEngine promises only if it's installed
+if [ $CFENGINE_INSTALLED -eq 0 ]
+then
+  echo -n "Configuring and installing initial CFEngine promises..."
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-init]: missing mkdir (l 292)

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**